subo/command/create_handler.go

```diff
@@ -0,0 +1,73 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/suborbital/atmo/directive"
+	"github.com/suborbital/subo/builder/context"
+	"github.com/suborbital/subo/subo/util"
+)
+
+type handlerData struct {
+	HandlerType string
+	Request     string
+	Method      string
+}
+
+func CreateHandlerCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "handler <name>",
+		Short: "create a new handler",
+		Long:  `create a new handler for Subo CLI`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			name := args[0]
```

Comment:
I was actually expecting `resource` to be the argument, i.e. `subo create handler /foo`. Could you explain why `name` was chosen?